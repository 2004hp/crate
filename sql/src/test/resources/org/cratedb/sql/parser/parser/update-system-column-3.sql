update t3 set foo='bar' where db.tablename."_version" = 'newest'