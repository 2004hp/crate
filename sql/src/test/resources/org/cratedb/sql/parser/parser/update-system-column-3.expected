org.cratedb.sql.parser.parser.UpdateNode@21d65a49
statementType: UPDATE
resultSet: 	
	org.cratedb.sql.parser.parser.SelectNode@7fc17ff9
	isDistinct: false
	resultColumns: 		
		org.cratedb.sql.parser.parser.ResultColumnList@6c46efab

		[0]:		
		org.cratedb.sql.parser.parser.ResultColumn@2520fb16
		exposedName: foo
		name: foo
		tableName: null
		isDefaultColumn: false
		type: null
		expression: 			
			org.cratedb.sql.parser.parser.CharConstantNode@36359f55
			value: bar
			type: CHAR(3) NOT NULL
		reference: 			
			org.cratedb.sql.parser.parser.ColumnReference@7a6fa140
			columnName: foo
			tableName: null
			type: null
	fromList: 		
		org.cratedb.sql.parser.parser.FromList@36c3af8c

		[0]:		
		org.cratedb.sql.parser.parser.FromBaseTable@693a2c6a
		tableName: t3
		updateOrDelete: UPDATE
		null
		correlation Name: null
		null
	whereClause: 		
		org.cratedb.sql.parser.parser.BinaryRelationalOperatorNode@6a3a191e
		operator: =
		methodName: equals
		type: null
		leftOperand: 			
			org.cratedb.sql.parser.parser.ColumnReference@c47370e
			columnName: _version
			tableName: db.tablename
			type: null
		rightOperand: 			
			org.cratedb.sql.parser.parser.CharConstantNode@176f7e6e
			value: newest
			type: CHAR(6) NOT NULL
targetTableName: 	
	org.cratedb.sql.parser.parser.TableName@e3f
	t3
